documentation/Installing-Clear-Containers-on-Ubuntu.md

```diff
@@ -43,7 +43,7 @@ cat << EOL | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
 [Service]
 ExecStart=
 ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/bin/cc-oci-runtime --default-runtime=cor
-EOL>>
+EOL
```

Comment:
This is not a line this is a file, I would suggest to change EOL by EOF to be aligned with bkms and best practices.